packages/cms-base/components/listing-filters/SwFilterProperties.vue

```diff
@@ -59,7 +59,7 @@ const toggle = () => {
             v-for="option in props.filter.options || props.filter.entities"
             :key="`${option.id}-${selectedOptionIds?.includes(option.id)}`"
             class="flex items-center"
-            @click="
+            @click.once="
```

Comment:
div's should not be clickable, it creates problem with accessibility. Could you try to make it on `change` event inside `input`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role will resolve accessibility problem

Reply:
or we can just react to `change` event of the input field and not have workarounds :)